tests/testthat/test-helper-rng.R

```diff
@@ -0,0 +1,5 @@
+# The default RNG kind changed in R > 3.6...the recorded tests
+# currently rely on randomness and are currently expecting
+# the default RNG kind from R 3.5
+# CRAN has suggested this as workaround for random tests
```

Comment:
It'll help in the future to have a sentence saying that we can get rid of this and update all the expected results when we're no longer testing with R >=3.5 on Travis.